buffer/src/main/java/io/netty/buffer/PooledByteBufAllocator.java

```diff
@@ -100,7 +100,21 @@ public void run() {
          *
          * See https://github.com/netty/netty/issues/3888.
          */
-        final int defaultMinNumArena = NettyRuntime.availableProcessors() * 2;
+        int defaultMinNumArena = NettyRuntime.availableProcessors() * 2;
```

Comment:
`defaultMinNumArena` is used for heap AND direct arenas, but IMO `MALLOC_ARENA_MAX` should only effect direct. Does that make sense?

Reply:
I am not sure... I think I would use it for both. Otherwise it might be envy more confusing.